### ISSUE TITLE
The  SErenet50 is less accurate than resnet50 model in imagenet?

### ISSUE THREAD

**Issue by sankin1770** [moskomule/senet.pytorch#23]

**Comment by moskomule** [comment]
No, SEResNet should be higher accuracy than ResNet. However, my hyper-parameter tuning was wrong and the result was bad.
